src/python/pants/backend/python/goals/tailor.py

```diff
@@ -166,41 +169,24 @@ def add_req_targets(files: Iterable[FileContent], alias: str, target_name: str)
                     )
                 )
 
-        def validate(path: str, contents: bytes, alias: str) -> bool:
+        def validate(path: str, contents: bytes, alias: str) -> None:
             if alias == "python_requirements":
                 return validate_python_requirements(path, contents)
             elif alias == "pipenv_requirements":
                 return validate_pipenv_requirements(contents)
             elif alias == "poetry_requirements":
                 return validate_poetry_requirements(contents)
-            return True
-
-        def validate_python_requirements(path: str, contents: bytes) -> bool:
-            try:
-                for _ in parse_requirements_file(contents.decode(), rel_path=path):
-                    pass
-            except Exception as e:
-                logger.warning(f"{e}")
-                return False
-            return True
-
-        def validate_pipenv_requirements(contents: bytes) -> bool:
-            try:
-                parse_pipenv_requirements(contents)
-                return True
-            except Exception as e:
-                logger.warning(f"{e}")
-                return False
-
-        def validate_poetry_requirements(contents: bytes) -> bool:
 
+        def validate_python_requirements(path: str, contents: bytes) -> None:
+            for _ in parse_requirements_file(contents.decode(), rel_path=path):
+                pass
```

Comment:
```suggestion
            parse_requirements_file(contents.decode(), rel_path=path):
```

Reply:
It's a generator, we need to consume its output somehow